2_feature_importance/2_feature_importance.ipynb

```diff
@@ -0,0 +1,500 @@
+{
```

Comment:
_Commented on notebook `2_feature_importance/2_feature_importance.ipynb` Cell 8 Line 1_
```markdown
The feature importance property of the *RandomForestRegressor* class is *feature_importances_*
```
You can hide the solution in the following way:
```
<details>
    <summary>Click to see the solution</summary>
    <p>Your solution goes here. It can be text, code, or any combination thereof.</p>
</details>
```

_Respond and view the context [here](https://gitnotebooks.com/oriol-pomarol/codegeo_workshops/pull/1)._

Reply:
Done for all exercises